src/components/DataElement/VoivodeshipsSplit.jsx

```diff
@@ -25,6 +23,7 @@ export default function VoivodeshipsSplit({ isLoading, data }) {
           <BarChart
             data={data}
             layout="vertical"
+            className="voivodeship-charts"
```

Comment:
I think this line can be replaced by the following code. Since we have Styletron installed we should take advantage of it. 
```suggestion
            className={css({
                        fontSize: '12px'
            })}
```